pkg/server/api/github.go

```diff
@@ -108,11 +106,16 @@ func (c *Api) fetchUserMappings() (map[string]string, error) {
 		variables["cursor"] = pageInfo.EndCursor
 	}
 
+	c.metrics.Sink.IncrCounter([]string{"github_fetches"}, 1)
```

Comment:
this could probably be constants defined at the top or even within the telemetry package